src/NCronJob/Configuration/Builder/IOptionChainerBuilder.cs

```diff
@@ -0,0 +1,14 @@
+namespace NCronJob;
+
+/// <summary>
+/// Expose a way to chain different options.
+/// </summary>
+public interface IOptionChainerBuilder
+{
+    /// <summary>
+    /// Chains another option to the job.
+    /// </summary>
+#pragma warning disable CA1716 // Identifiers should not match keywords
```

Comment:
If you want, put this into the `.editorconfig`

It is not valid for the whole project, not just this place.